data/convert.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+import pytz
+
+
+def convert(date):
+    """
+    Convert string date to Django timezone(aware datetime)
+
+    :param date: a string representing datetime
+                 e.g. "2016-08-19T11:38:15" or "2017-11-23T17:50:14.035"
+    :return: a Django timezone(aware datetime object)
+    """
+    _date = date.split('.')[0]
```

Comment:
underscore prefixes are not useful in local variables.

and this one , and the algorithm here, is not explained anywhere.

I guess it is removing milliseconds.

make that clear in the variable names and in the docstring.